code/src/Components/AlbumCover.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import Radium from 'radium'
+import styles from '../styles'
+import AlbumCoverBackground from './AlbumCoverBackground'
+import AlbumCoverIcons from './AlbumCoverIcons'
+
+const AlbumCover = (props) => {
+  const [hoverBackgroundVisible, setHoverBackgroundVisible] = useState(false)
+  const [underline, setUnderline] = useState(false)
+  return (
+    <div>
+      <div style={styles.imgWrapper} onMouseEnter={() => setHoverBackgroundVisible(true)} onMouseLeave={() => setHoverBackgroundVisible(false)}>
+        <a href={props.linkAlbum}>
```

Comment:
nice solution with onMouseEnter and onMouseLeave. We expected you to stick to the hover effect in css as we've done before. But this also works. 😎